_extensions/posit-docs/theme.scss

```diff
@@ -80,6 +80,13 @@ $list-group-color: $primary !default;
     font-weight: 500;
 }
 
+/* Style for the version included in the website title */
+.navbar-title small {
+    font-size: 0.6em;
+    display: block;
+    padding-left: 1em;
+}
+
```

Comment:
This doesn't meet accessibility standards. 
If you remove this and add it to an issue, I can play around with this when I am working through some of the dark theme changes.

Reply:
Adjusted.